draft-ietf-openpgp-pqc.md

```diff
@@ -936,6 +936,12 @@ If the pair `AES-256` with `OCB` is not explicitly in the list of the "Preferred
 This is justified since `AES-256` and `OCB` are mandatory to implement.
 If the pair `AES-128` with `OCB` is also implicitly added to the list, it is added after the pair `AES-256` with `OCB`.
 
+When encrypting to a certificate that contains a PQ(/T) key
+and if one of the subpackets "Preferred Symmetric Ciphers for v1 SEIPD" or "Preferred AEAD Ciphersuites" is missing,
+an implementation SHOULD treat this as if the subpacket with an empty list exists
+and implicitly add algorithms as describhed in this section.
+Support for the v2 SEIPD feature MUST NOT be assumed on the basis of implicitly added AEAD algorithm pairs alone.
```

Comment:
this final line is passive voice: who is doing the assuming or not assuming?

How about being clearer, like: 

> When encrypting to a OpenPGP certificate that contains a PQ(/T) key, an implementation MUST NOT infer that the certificate's owner supports v2 SEIPD solely on the basis of the implicitly added AEAD algorithm pairs.

Reply:
The bottom line seems to be that there is consenus to only add the sentence that @dkg propsed above. In that case I ask @TJ-91 to update the PR accordingly. I am ready to close issue #106, which was the reason for the initially proposed changes in this PR, that was raised by me as irrelevant.

Reply:
@wussler @falko-strenzke I rebased now with only that change